### DISCUSSION TITLE
How to avoid css strip in debug?

### DISCUSSION THREAD

**Discussion by Gambero81** [tailwindlabs/tailwindcss#8698] [Ideas]
Hi, 

Using the `content` section of `tailwind.config.js` we can avoid to generate unused css class and this is great for production.  

Now i would like to disable this feature in a debug environment so i can apply classes on the fly (for example in chrome dev tools).

There is a way to do this?

I noted the `safelist` section, but how can tell in this section to consider all classes?

Thanks

**Comment by asri134** [comment]
You can automatically remove empty tags, combine nested font tags, and otherwise improve messy or unreadable HTML or XHTML code.

**Comment by Gambero81** [comment]
this is the answer i needed: https://github.com/tailwindlabs/tailwindcss/discussions/6557
